docs/src/interface.md

```diff
@@ -1,22 +1,35 @@
 # The AbstractSciMLOperator Interface
 
-## Formal Properties of DiffEqOperators
+## Formal Properties of SciMLOperators
 
 These are the formal properties that an `AbstractSciMLOperator` should obey
 for it to work in the solvers.
 
-## AbstractDiffEqOperator Interface Description
+1. An `AbstractSciMLOperator` represents a linear or nonlinear operator with input/output
+   being `AbstractArray`s. Specifically, a SciMLOperator, `L`, of size `(M,N)` accepts
+   input argument `u` with leading length `N`, i.e. `size(u, 1) == N`, and returns an
+   `AbstractArray` of the same dimension with leading length `M`, i.e. `size(L * u, 1) == M`.
+   Internally, `L` lazily reshapes `u` to a matrix of size `(N, length(u) \div N)` and
+   independently acts on the column-vectors. The reshape operation is skipped for
```

Comment:
No, a counter example of this is derivative operations and convolutions. Columns are not independent. It's a linear operator on the whole `vec(u)`.